test_util/aws.py

```diff
@@ -55,8 +55,11 @@ def fetch_stack(stack_name, boto_wrapper):
         if resource.logical_resource_id == 'MasterServerGroup':
             log.debug('Using Basic DC/OS Cloudformation interface')
             return DcosCfStack(stack_name, boto_wrapper)
-    log.debug('Using VPC Cloudformation interface')
-    return BareClusterCfStack(stack_name, boto_wrapper)
+        if resource.logical_resource_id == 'BareServerAutoScale':
+            log.debug('Using Bare Cluster Cloudformation interface')
+            return BareClusterCfStack(stack_name, boto_wrapper)
+    log.warning('No recognized resources found; using generic stack')
+    return CfStack(stack_name, boto_wrapper)
```

Comment:
Question (not a blocker) - How does this change in behavior affects the caller (or the methods that currently call fetch_stack)?